### ISSUE TITLE
Cannot create sample table

### ISSUE THREAD

**Issue by koksen** [SnappyDataInc/snappydata#1251]
Hi,

I downloaded the enterprice version from the homepage. 

On my local machine I does this: 
val sample = snappy.createSampleTable("interactions_stratified", Option("INTERACTIONS"),  interactionSchema, Map(("fraction", " 0.01")), true)

Then I get this error:
19/01/31 14:46:59 INFO SnappyContext: Initializing SnappyData in cluster mode: Local mode: sc = org.apache.spark.SparkContext@36068727, url = mcast-port=0
Exception in thread "main" java.lang.ClassNotFoundException: Failed to find data source: org.apache.spark.sql.sampling.DefaultSource. Please find packages at http://spark.apache.org/third-party-projects.html
	at org.apache.spark.sql.execution.datasources.DataSource$.lookupDataSource(DataSource.scala:594)

Seems that I have not loaded the datasource to support this method. What do I need to put in my sbt?

Current:
libraryDependencies += "io.snappydata" %% "snappydata-cluster" % "1.0.2.1" from "file:///lib/snappydata-cluster_2.11-1.0.2.1.jar"

**Comment by guanxianchun** [comment]
I have the same problem. Have you solved your problem

**Comment by koksen** [comment]
No, I couldn’t get an answer or find anything in the open sourced code. 
We wrote our own stuff instead

**Comment by ahshahid** [comment]
Will check and get back to you. The aqp jar should have been there in
enterprise build.

**Comment by ahshahid** [comment]
Can you try with putting following dependency also?

libraryDependencies += "io.snappydata" %% "snappydata-aqp" % "1.0.2.1" from
"file:///lib/snappydata-aqp_2.11-1.0.2.1.jar"

Also pls verify that you have snappydata-aqp_2.11-1.0.2.1.jar in the lib
directory.

Regards
Asif
